seed/utils/labels.py

```diff
@@ -17,8 +17,37 @@
 )
 
 
-def _get_labels(request, qs, super_organization, inv_type):
-    inventory = InventoryFilterBackendWithInvType().filter_queryset_with_inv(
+def filter_labels_for_inv_type(request, inv_type):
+    """
+    Method used to filter labels by inventory type and return is_applied inventory id's
+    Method was initially built as a class defined function above to handle more parameters.
+    """
+    params = request.query_params.dict()
+    # Since this is being passed in as a query string, the object ends up
+    # coming through as a string.
+    params['filter_params'] = json.loads(params.get('filter_params', '{}'))
+    inventory_type = inv_type
```

Comment:
Let's remove this extra variable assignment and use inv_type directly below. We could also just rename the argument to be "inventory_type", and delete this line.